src/storage/src/hummock/shared_buffer/shared_buffer_batch.rs

```diff
@@ -856,13 +662,66 @@ impl<D: HummockIteratorDirection> HummockIterator for SharedBufferBatchIterator<
 }
 
 pub struct SharedBufferDeleteRangeIterator {
-    inner: Arc<SharedBufferBatchInner>,
+    monotonic_tombstone_events: Vec<MonotonicDeleteEvent>,
     next_idx: usize,
 }
 
 impl SharedBufferDeleteRangeIterator {
-    pub(crate) fn new(inner: Arc<SharedBufferBatchInner>) -> Self {
-        Self { inner, next_idx: 0 }
+    pub(crate) fn new(
```

Comment:
The logic is moved from `SharedBufferBatchInner::new`. This method is only used in test.

Reply:
If it is only used in test, should we add `#[cfg(any(test, feature = "test"))]` ?